packages/api-providers/src/contracts/abis/BadRecipient.json

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
Is this file needed?

Reply:
Will sync with @nathan on the integration of `@webb-tools/contracts`